redwood-treehouse/src/hostMain/kotlin/app/cash/redwood/treehouse/TreehouseApp.kt

```diff
@@ -111,15 +110,7 @@ public class TreehouseApp<A : Any> internal constructor(
       viewToBoundContent.remove(view)
     }
 
-    val ziplineSession = this.ziplineSession
-    if (ziplineSession != null) {
-      ziplineSession.bind(view, content)
-    } else {
-      // If we're waiting for code to load, show a loading indicator until it's ready.
```

Comment:
I don’t love the slight regression here . . . the caller doesn’t know if code is already ready, so they potentially waste work creating spinners & placeholders that aren’t necessary

Reply:
Ah, that makes sense. We can keep it for that.

Reply:
#566